bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -38,6 +39,11 @@ limitations under the License.
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.apache.logging.log4j</groupId>
+        <artifactId>log4j-api</artifactId>
```

Comment:
what about log4j-core?

Reply:
I think it only has log4j-api dependency here, log4j-api should be included in the banned dependencies